### PR TITLE
Continuation of exception message review

### PR DESCRIPTION
**Pull request by smitpatel**
Part of #7201

### CODE REVIEW

File: src/EFCore/Properties/CoreStrings.resx [hunk 1]
```diff
@@ -289,7 +289,7 @@
     <value>Cannot access a disposed context instance. A common cause of this error is disposing a context instance that was resolved from dependency injection and then later trying to use the same context instance elsewhere in your application. This may occur if you are calling 'Dispose' on the context instance, or wrapping it in a using statement. If you are using dependency injection, you should let the dependency injection container take care of disposing context instances.</value>
   </data>
   <data name="ConventionsInfiniteLoop" xml:space="preserve">
-    <value>The convention invocations have reached the recursion limit. This is likely an issue in Entity Framework, please report it.</value>
+    <value>The convention invocations have reached the recursion limit. This is likely an issue in EF Core, please file an issue at https://go.microsoft.com/fwlink/?linkid=2142044.</value>
```
Comment (AndriySvyryd) [inline src/EFCore/Properties/CoreStrings.resx:292]: We are actually changing everything to "Entity Framework", see https://github.com/dotnet/efcore/issues/7201

Comment (smitpatel) [reply src/EFCore/Properties/CoreStrings.resx:292]: So @roji will update this when submitting a PR? 🤔

Comment (AndriySvyryd) [reply src/EFCore/Properties/CoreStrings.resx:292]: Yes, but don't make it harder 😄

**Review by AndriySvyryd — APPROVED** [review]

File: src/EFCore/Properties/CoreStrings.resx (hunk 1 above)
Comment (smitpatel) [reply src/EFCore/Properties/CoreStrings.resx]: Added another commit which fixed it for whole solution.

**Review by AndriySvyryd — APPROVED** [review]

**Comment by AndriySvyryd** [comment]
@Pilchie For RC2

**Comment by Pilchie** [comment]
Approved for .NET 5.0 RC2.

**Pull request merged**